muffnn/mlp/base.py

```diff
@@ -112,6 +112,9 @@ def partial_fit(self, X, y, monitor=None, **kwargs):
         X, y = self._check_inputs(X, y)
         assert self.batch_size > 0, "batch_size <= 0"
 
+        if self.transform_layer is None:
+          self._transform_layer = len(self.hidden_units)
+
```

Comment:
Nope. This needs to go in the block right below this. Also, you need to set `_transform_layer` even if `transform_layer` is not None.